docs/protocol/papers/protocol-roadmap.md

```diff
@@ -8,23 +8,23 @@ If you have feature requests or feedback, we want to hear from you. Join us on [
 
 ## Denial of service mitigation
 
-* Explicit limits on number and size of blockchain entities (size of the blocks, number of transactions etc).
-* Fine-tuned runtime cost limits for the control and issuance programs.
-* Improvements to consensus algorithm.
+* Fine-tuned and tested runtime costs for the virtual machine
+* Adjustable run limit, as well as adjustable limits on the size of blocks, transactions, programs, and other data structures
+* Improved consensus algorithms to guarantee liveness under weaker assumptions
 
 ## Privacy
 
-* Homomorphically encrypted asset identifiers and amounts to provide secrecy for balances and financial parameters of the transactions.
-* Controlled traceability of the transactions; hiding the link between transaction inputs and the previous transactions’ outputs.
+* Confidential transactions that hide assets and amounts through homomorphic encryption
+* Unlinkable transactions that further hide the identities of transacting parties
```

Comment:
Not unlinkable, but untraceable. Unlinkability refers to links between outputs and accounts. I'd like to keep using these terms consistently with existing research.